### ISSUE TITLE
[new-feature] SD-WAN member status Trigger

### ISSUE THREAD

**Issue by ugoviti** [mbdraks/fortinet-zabbix#30]
Hi Michel, thank you very much for your really good Fortigate Template! it's the best one!

I added a trigger prototype for the SD-WAN on my local copy, If you want include upstream, this is the code (change the name and descriptions as you want of course):

```
                            <trigger_prototypes>
                                <trigger_prototype>
                                    <expression>{last(#1)}=1</expression>
                                    <name>SD-WAN status of {#HC_NAME} for member {#MEMBER_SEQ}</name>
                                    <opdata>Status: {ITEM.LASTVALUE1}</opdata>
                                    <priority>DISASTER</priority>
                                </trigger_prototype>
                            </trigger_prototypes>
```
PS. would be great also a Interface Bandwidth trigger, to get warned when a Interface bandwidth exceeds a threshold. When I find some time I'll works on it

Keep up the good work.

Kind regards

**Comment by mbdraks** [comment]
Tks for the feedback! I'll include it in the next update.

Keeping this issue open until release.

**Comment by mbdraks** [comment]
Closed by #31
